.github/workflows/CICD.yml

```diff
@@ -0,0 +1,21 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
+    
+    runs-on: ubuntu-latest
+  
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js
+      uses: actions/setup-node@v3
+      with:
+        # node-version: ${{ matrix.node-version }}
```

Comment:
I don't know the importance of the node version here, but why is this commented out?

Reply:
Because we don't need to require what version of node.js we are using, but if you think we need I can set it, so I commented on it here